lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -1589,9 +1610,26 @@ std::istream &operator>>(std::istream &I, SPIRVModule &M) {
   }
 
   Decoder >> MI.SPIRVVersion;
-  if (!M.getErrorLog().checkError(MI.SPIRVVersion <= SPV_VERSION,
-                                  SPIRVEC_InvalidModule,
-                                  "unsupported SPIR-V version number")) {
+  bool SPIRVVersionIsKnown =
+      static_cast<uint32_t>(VersionNumber::MinimumVersion) <= MI.SPIRVVersion &&
+      MI.SPIRVVersion <= static_cast<uint32_t>(VersionNumber::MaximumVersion);
+  if (!M.getErrorLog().checkError(
+          SPIRVVersionIsKnown, SPIRVEC_InvalidModule,
+          "unknown SPIR-V version number. Range of supported/known SPIR-V "
```

Comment:
I think it might be useful to know what the actual version is. Please print value of `MI.SPIRVVersion` to the error message.

Reply:
Done